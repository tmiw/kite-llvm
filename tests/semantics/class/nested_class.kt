class X [
    "initializing X"|print,

    class Y [
        "initializing Y"|print
    ]
];

"inside main program"|print;
