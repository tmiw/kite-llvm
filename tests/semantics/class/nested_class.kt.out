initializing X
initializing Y
inside main program
